Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -42,8 +42,18 @@ final class PackageToolTests: CommandsTestCase {
     }
 
     func testUsage() throws {
-        let stdout = try execute(["-help"]).stdout
-        XCTAssertMatch(stdout, .contains("USAGE: swift package"))
+        do {
+            let stdout = try execute(["-help"]).stdout
+        } catch let error as SwiftPMProductError {
+            switch error {
+            case let .executionFailure(_, _, stderr):
+                XCTAssertMatch(stderr, .contains("Usage: swift package"))
+            default:
+                throw error
+            }
+        } catch {
+            throw error
+        }
```

Comment:
nit: this whole block can be made more concise

Reply:
maybe something like this? I also added an `XCTFail` for when it does not throw since that seems to be an error case? you could also use `XCTAssertThrowsError` for that.

```
do {
            let stdout = try execute(["-help"]).stdout
            XCTFail("expecting an error")
         } catch SwiftPMProductError.executionFailure(_, _, let stderr)  {
             XCTAssertMatch(stderr, .contains("Usage: swift package"))
         }
```